tests/PKSim.Tests/Core/OriginDataMapperSpecs.cs

```diff
@@ -125,6 +125,75 @@ public void should_have_mapped_the_value_origin()
       }
    }
 
+   public class When_mapping_an_origin_data_to_snapshot_where_age_and_weight_are_equal_to_the_default_value : concern_for_OriginDataMapper
+   {
+      protected override async Task Context()
+      {
+         await base.Context();
+         _species.AddPopulation(_anotherPopulation);
+         _speciesPopulation.AddGender(_anotherGender);
+
+         A.CallTo(() => _individualModelTask.MeanAgeFor(_originData)).Returns(DomainHelperForSpecs.ConstantParameterWithValue(_originData.Age.Value));
+         A.CallTo(() => _individualModelTask.MeanWeightFor(_originData)).Returns(DomainHelperForSpecs.ConstantParameterWithValue(_originData.Weight));
+      }
+
+      protected override async Task Because()
+      {
+         _snapshot = await sut.MapToSnapshot(_originData);
+      }
+
+      [Observation]
+      public void should_save_the_value_for_height_and_gestational_age()
+      {
+         _snapshot.Species.ShouldBeEqualTo(_originData.Species.Name);
+         _snapshot.Population.ShouldBeEqualTo(_originData.SpeciesPopulation.Name);
+         _snapshot.Gender.ShouldBeEqualTo(_originData.Gender.Name);
+
+         _snapshot.Height.ShouldBeEqualTo(_heightSnapshotParameter);
+         _snapshot.GestationalAge.ShouldBeEqualTo(_gestationalAgeSnapshotParameter);
+      }
+
+      [Observation]
+      public void should_not_save_the_value_for_age_and_weight()
+      {
+         _snapshot.Age.ShouldBeNull();
+         _snapshot.Weight.ShouldBeNull();
+      }
+   }
+
+   public class When_mapping_an_origin_data_to_snapshot_where_age_ga_height_and_weight_are_equal_to_default_value : concern_for_OriginDataMapper
+   {
+      protected override async Task Context()
+      {
+         await base.Context();
+         _species.AddPopulation(_anotherPopulation);
+         _speciesPopulation.AddGender(_anotherGender);
+
+         A.CallTo(() => _individualModelTask.MeanAgeFor(_originData)).Returns(DomainHelperForSpecs.ConstantParameterWithValue(_originData.Age.Value));
```

Comment:
Funny enough, the way back was already implemented so that we could set a default value if a snapshot value was missing